packages/sdk/package.json

```diff
@@ -55,9 +55,12 @@
     "eth-rpc-errors": "^4.0.3",
     "eventemitter2": "^6.4.7",
     "extension-port-stream": "^2.0.1",
+    "i18next": "22.5.1",
```

Comment:
We can probably skip adding direct dependency as it should be retreives from sdk-install-modal-web